snapshot/snapshot.go

```diff
@@ -98,6 +93,22 @@ func NewSnapshotter(ctx context.Context, cfg *config.SnapshotterConfig) (snapsho
 		}
 	}
 
+	var skipSSLVerify bool
+	var daemonConfig *daemonconfig.DaemonConfig
+	fsDriver := config.GetFsDriver()
+	if fsDriver == config.FsDriverFscache || fsDriver == config.FsDriverFusedev {
+		config, err := daemonconfig.NewDaemonConfig(config.GetFsDriver(), cfg.DaemonConfig.NydusdConfigPath)
+		if err != nil {
+			return nil, errors.Wrap(err, "load daemon configuration")
+		}
+		daemonConfig = &config
+		_, backendConfig := config.StorageBackend()
+		skipSSLVerify = backendConfig.SkipVerify
+	} else {
+		// TODO: config skipSSLKVerify for tarfs etc
+		skipSSLVerify = true
```

Comment:
Why the option can't be received from configuration?

Reply:
My bad, it should be
```
@@ -105,8 +105,7 @@ func NewSnapshotter(ctx context.Context, cfg *config.SnapshotterConfig) (snapsho
                _, backendConfig := config.StorageBackend()
                skipSSLVerify = backendConfig.SkipVerify
        } else {
-               // TODO: config skipSSLKVerify for tarfs etc
-               skipSSLVerify = true
+               skipSSLVerify = config.GetSkipSSLVerify()
        }
 
        var blockdevManager *mgr.Manager

```